mmcv/cnn/bricks/padding.py

```diff
@@ -8,7 +10,7 @@
 PADDING_LAYERS.register_module('replicate', module=nn.ReplicationPad2d)
 
 
-def build_padding_layer(cfg, *args, **kwargs):
+def build_padding_layer(cfg: Optional[Dict], *args, **kwargs) -> nn.Module:
```

Comment:
It seems the `cfg` must be a dict. Suggest to modify the docstring and type hints